### DISCUSSION TITLE
feature: A common bank for all characters on an account.

### DISCUSSION THREAD

**Discussion by Gorgaki** [AscensionGameDev/Intersect-Engine#2347] [Ideas]
### Description

I'd like to share an idea with you. A feature that could be useful in the game design of many games is the possibility of offering the choice of making a bank common to all characters on the same account. This could be very useful for mmorpg replayability. Gameplay loops are generally based around collecting or looting resources or items, which can be very off-putting and discourage players from playing multiple characters.

### Intended Use-Case

Propose the bank in the events editor for one character or for all the characters in the account.

### Duplicate Check

- [X] This feature request is not a duplicate to the best of my knowledge.

**Comment by dhmello** [comment]
I agree, but there should be a CHARACTER BANK and an ACCOUNT bank, because there are some items that I would not like to be shared with the characters on the account, such as starting items (potions, for example). The player would create a bunch of characters and save them in the account bank.

There should be a limiter on the item, for example:

Can User Bank
Can Private Bank
